interface/code_systems/dataloads_ajax.php

```diff
@@ -229,7 +229,7 @@
                 success: function(data) {
                     $(stg_load_id).hide();
                     $(stg_dets_id).html(data);
-                    $(document).ready(function() {});
+                    $(function() {});
```

Comment:
This seems odd that there is no function in here. I can't think of any reason to do that? Unless I am missing something.